docker-for-mac/apple-m1.md

```diff
@@ -1,16 +1,64 @@
 ---
-description: Apple M1 Tech Preview
+description: Apple M1
 keywords: Docker Desktop, M1, Silicon, Apple, tech preview, dev preview
-title: Apple M1 Tech Preview
+title: Apple M1
 toc_min: 2
 toc_max: 3
 ---
 
-Welcome to the tech preview of Docker Desktop for Apple M1. This tech preview is aimed at early adopters of Apple M1 machines, who would like to try an experimental build of Docker Desktop.
+Welcome to Docker Desktop for Apple Silicon.
 
 > **Note**
 >
-> Docker Desktop on Apple M1 chip is still under development. We recommend that you do not use tech preview builds in production environments.
+> Docker Desktop for Apple Silicon is still under development. Please try these builds and report issues on [docker/for-mac](https://github.com/docker/for-mac).
```

Comment:
How about "We encourage you to try these builds and report any issues on [docker/for-mac](https://github.com/docker/for-mac) GitHub repository."